src/Nancy.Authentication.Forms.Tests/FormsAuthenticationFixture.cs

```diff
@@ -162,7 +161,71 @@ public void Should_return_ok_response_when_user_logs_in_without_redirect()
             result.StatusCode.ShouldEqual(HttpStatusCode.OK);
         }
 
-        [Fact]
+		#region Throw helpful exception when the configuration is not enabled

```

Comment:
Ug. Yuck. Spaces.  Ok.  There isn't a style-guide, is there?  There seems to be a space for one, but no link.